src/index.html

```diff
@@ -11,11 +11,11 @@
     <link rel="stylesheet" href="./reset.css">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
+  <body class="page">
```

Comment:
Don't set classes on body element. This is redundant as there is no possibility for you to have more than one `body` element. If you must target body in css do it with tag, however I assume you want to have some type of container for these forms, I suggest wrapping them in a div with a class `wrapper`.